src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -769,38 +747,125 @@ class ContentSidebar extends PureComponent<Props, State> {
         });
     };
 
+    /**
+     * Sorts and formats assignments, and then adds them to their task. 
+     *
+
+     * @param {Task} task - Task to which the assignments belong
+     * @param {Task} assignments - List of task assignments
+     * @return {Task}
+     */
+    appendAssignmentsToTask(task: Task, assignments: Array<TaskAssignment>) {
+        if (!assignments) {
```

Comment:
Would it make sense to change to ```assignments?``` since it seems to be optional?

Reply:
I took the negative to avoid the indentation